src/ContainerAppHelper.ts

```diff
@@ -0,0 +1,434 @@
+     * @returns true if the Container App exists, false otherwise
+     */
+    public async doesContainerAppExist(containerAppName: string, resourceGroup: string): Promise<boolean> {
+        toolHelper.writeDebug(`Attempting to determine if Container App with name "${containerAppName}" exists in resource group "${resourceGroup}"`);
+        try {
+            let command = `az containerapp show -n ${containerAppName} -g ${resourceGroup} -o none`;
+            let executionResult = await util.executeAndThrowIfError(command);
+            return executionResult.exitCode === 0;
+        } catch (err) {
+            toolHelper.writeWarning(err.message);
+            return false;
+        }
+    }
+
+    /**
+     * Determines if the provided Container App Environment exists in the provided resource group.
+     * @param containerAppEnvironment - the name of the Container App Environment
+     * @param resourceGroup - the resource group that the Container App Environment is found in
+     * @returns true if the Container App Environment exists, false otherwise
+     */
+    public async doesContainerAppEnvironmentExist(containerAppEnvironment: string, resourceGroup: string): Promise<boolean> {
+        toolHelper.writeDebug(`Attempting to determine if Container App Environment with name "${containerAppEnvironment}" exists in resource group "${resourceGroup}"`);
+        try {
+            let command = `az containerapp env show -n ${containerAppEnvironment} -g ${resourceGroup} -o none`;
+            let executionResult = await util.executeAndThrowIfError(command);
+            return executionResult.exitCode === 0;
+        } catch (err) {
+            toolHelper.writeWarning(err.message);
+            return false;
+        }
+    }
+
+    /**
+     * Determines if the provided resource group exists.
+     * @param resourceGroup - the name of the resource group
+     * @returns true if the resource group exists, false otherwise
+     */
+    public async doesResourceGroupExist(resourceGroup: string): Promise<boolean> {
+        toolHelper.writeDebug(`Attempting to determine if resource group "${resourceGroup}" exists`);
+        try {
+            let command = `az group show -n ${resourceGroup} -o none`;
+            let executionResult = await util.executeAndThrowIfError(command);
+            return executionResult.exitCode === 0;
+        } catch (err) {
+            toolHelper.writeWarning(err.message);
+            return false;
+        }
+    }
+
+    /**
+     * Gets the default location for the Container App provider.
+     * @returns the default location if found, otherwise 'eastus2'
+     */
+    public async getDefaultContainerAppLocation(): Promise<string> {
+        toolHelper.writeDebug(`Attempting to get the default location for the Container App service for the subscription.`);
+        try {
+            let command = `az provider show -n Microsoft.App --query \"resourceTypes[?resourceType=='containerApps'].locations[] | [0]\"`
```

Comment:
nit: do we need to escape the double quotes in this string? I'm afraid that it'll interpreted as `\"` instead of `"` since we're using the tick mark to wrap the entire string variable rather than double quotes

Reply:
Updated.